neqo-transport/src/cc/classic_cc.rs

```diff
@@ -1045,42 +1048,37 @@ mod tests {
         assert_eq!(cc.acked_bytes, 0);
 
         // Now we are in the congestion avoidance state.
-        let mut pkts = Vec::new();
-        for i in 0..CWND_PKTS_CA {
-            let p = SentPacket::new(
-                PacketType::Short,
-                u64::try_from(i + 3).unwrap(), // pn
-                now(),                         // time sent
-                true,                          // ack eliciting
-                Vec::new(),                    // tokens
-                MAX_DATAGRAM_SIZE,             // size
-            );
-            cc.on_packet_sent(&p);
-            pkts.push(p);
-        }
-        assert_eq!(cc.bytes_in_flight(), CWND_INITIAL / 2);
-
-        for i in 0..CWND_PKTS_CA - 2 {
-            cc.on_packets_acked(&pkts[i..=i], RTT, now());
-
-            assert_eq!(
-                cc.bytes_in_flight(),
-                (CWND_PKTS_CA - i - 1) * MAX_DATAGRAM_SIZE
-            );
-            assert_eq!(cc.cwnd(), CWND_PKTS_CA * MAX_DATAGRAM_SIZE);
-            assert_eq!(cc.acked_bytes, MAX_DATAGRAM_SIZE * (i + 1));
-        }
-
-        // Now we are app limited
-        for i in CWND_PKTS_CA - 2..CWND_PKTS_CA {
-            cc.on_packets_acked(&pkts[i..=i], RTT, now());
-
+        assert_eq!(cc.state, State::CongestionAvoidance);
+        // simulate 10 packet bursts below app_limit
+        for k in 0..10 {
+            // always stay below app_limit during sent.
+            let mut pkts = Vec::new();
+            for i in 0..BELOW_APP_LIMIT_PKTS {
+                let p = SentPacket::new(
+                    PacketType::Short,
+                    u64::try_from(k * BELOW_APP_LIMIT_PKTS + i + 3).unwrap(), // pn
+                    now() + (k as u32 + 2) * RTT,                             // time sent
```

Comment:
k is usize, so maybe

```suggestion
                    now() + (u32::try_from(k).unwrap() + 2) * RTT,                             // time sent
```

Reply:
Alternatively, to save on duplication, you might want to make a variable for this, so that you can just do:

```rust
let mut now = now();
// ...
for k in 0..10 {
now += RTT;
for i in 0..BELOW_APP_LIMIT_PKTS {
}
now += RTT;
}
```